app/src/main/java/an/maguste/android/navier/storage/repository/MoviesRepositoryImpl.kt

```diff
@@ -32,4 +33,13 @@ class MoviesRepositoryImpl : MoviesRepository {
         moviesDB.moviesDao().getAll().map { MovieMapper.toMovieDomain(it) }
     }
 
+
+    /** get concrete movie */
+    override suspend fun getMovieById(id: Long): Movie? = withContext(Dispatchers.IO) {
+        val movie = moviesDB.moviesDao().getMovie(id)
+
+        if (movie != null){
+            MovieMapper.toMovieDomain(movie)
+        } else null
```

Comment:
Эквивалентно такой записи
```suggestion
        movie?.let { MovieMapper.toMovieDomain(it) }
```

Reply:
Верно, забыла привести код в порядок после экспериментов. Спасибо)